src/Microsoft.Identity.Web.TokenAcquisition/MergedOptions.cs

```diff
@@ -215,7 +215,13 @@ internal static void UpdateMergedOptionsFromMicrosoftIdentityOptions(MicrosoftId
             mergedOptions.TimeProvider = microsoftIdentityOptions.TimeProvider;
 #endif
 #if NET9_0_OR_GREATER
-            mergedOptions.AdditionalAuthorizationParameters = microsoftIdentityOptions.AdditionalAuthorizationParameters;
+            if (microsoftIdentityOptions.AdditionalAuthorizationParameters != null)

```

Comment:
Is there a change in net9 that necessitates it being done this way?

Reply:
@westin-m This is a new property in NET9 and seems that Id Web is using it as a read-only property so, needed to extract each value individually.